### PR TITLE
implement penalty for calibration

### PR DESCRIPTION
**Pull request by AurelienRenou**
implement function to put penalty for calibrate support in ulna and forces the system to act in a realistic way

### CODE REVIEW

File: models/enums.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    KINOVA_MERGE = str(Path("KINOVA_merge.bioMod").absolute())
+    KINOVA_MERGE_INVERSE_KINEMATICS = str(Path("KINOVA_merge_inverse_kinematics.bioMod").absolute())
+    KINOVA_MERGE_WITHOUT_FLOATING_BASE_TEMPLATE = str(Path("KINOVA_merge_without_floating_base_template.bioMod").absolute())
+    KINOVA_MERGE_WITHOUT_FLOATING_BASE_WITH_VARIABLES = str(Path("KINOVA_merge_without_floating_base_template_with_variables.bioMod").absolute())
+    KINOVA_MERGE_WITHOUT_FLOATING_BASE_WITH_6_DOF_SUPPORT_TEMPLATE = str(Path("KINOVA_merge_without_floating_base_with_6_dof_support_template.bioMod").absolute())
+    KINOVA_MERGE_WITHOUT_FLOATING_BASE_WITH_6_DOF_SUPPORT_VARIABLES = str(Path("KINOVA_merge_without_floating_base_with_6_dof_support_template_with_variables.bioMod").absolute())
+    KINOVA_MERGE_WITHOUT_FLOATING_BASE_WITH_ROTOTRANS_SUPPORT_TEMPLATE = str(Path("KINOVA_merge_without_floating_base_with_rototrans_template.bioMod").absolute())
+    KINOVA_MERGE_WITHOUT_FLOATING_BASE_WITH_ROTOTRANS_SUPPORT_VARIABLES = str(Path("KINOVA_merge_without_floating_base_with_rototrans_template_with_variables.bioMod").absolute())
```
Comment (Ipuch) [inline models/enums.py:20]: Instead of KINOVA_MERGE, write WU_AND_KINOVA and place this category after Wu models in another section.

Comment (Ipuch) [reply models/enums.py:20]: Group them by two, template and variables and add space after a group

File: models/enums.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+    WU = str(Path("wu_converted_definitif.bioMod").absolute())
+    WU_INVERSE_KINEMATICS = str(Path("wu_converted_definitif_inverse_kinematics.bioMod").absolute())
+    WU_INVERSE_KINEMATICS_XYZ = str(Path("wu_converted_definitif_inverse_kinematics_XYZ.bioMod").absolute())
+    WU_WITHOUT_FLOATING_BASE = str(Path("wu_converted_definitif_without_floating_base.bioMod").absolute())
+    WU_WITHOUT_FLOATING_BASE_TEMPLATE = str(Path("wu_converted_definitif_without_floating_base_template.bioMod").absolute())
+    WU_WITHOUT_FLOATING_BASE_TEMPLATE_FIXED = str(Path("wu_converted_definitif_without_floating_base_template_fixed.bioMod").absolute())
+    WU_WITHOUT_FLOATING_BASE_TEMPLATE_QUAT = str(Path("wu_converted_definitif_without_floating_base_template_quat.bioMod").absolute())
```
Comment (Ipuch) [inline models/enums.py:28]: _Template has to be the last key word

File: models/enums.py [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    WU_WITHOUT_FLOATING_BASE_VARIABLES = str(Path("wu_converted_definitif_without_floating_base_template_with_variables.bioMod").absolute())
+    WU_WITHOUT_MODIF = str(Path("wu_converted_definitif_without_modif.bioMod").absolute())
```
Comment (Ipuch) [inline models/enums.py:30]: What is the difference with   WU = str(Path("wu_converted_definitif.bioMod").absolute()) ?

File: kinova_tracking/pos_kinova_arm_2.py [hunk 4]
```diff
@@ -13,19 +13,18 @@
...
     ----------
-    labels_markers_list : list[str]
-        List of markers labels
+    marker_to_move: int
+        indices of the marker to move
     c3d_file : c3d
         c3d file to move the markers.
-    offset : float
-        Offset to apply to the markers in mm.
+    offset : list[int]
```
Comment (Ipuch) [inline kinova_tracking/pos_kinova_arm_2.py:26]: why not a numpy array ?

File: kinova_tracking/pos_kinova_arm_2.py [hunk 5]
```diff
@@ -13,19 +13,18 @@
 import random
 
 
-def move_marker_table(labels_markers_list: list[str], c3d_file: c3d, offset: float) -> np.array:
+def move_marker(marker_to_move: str, c3d_file: c3d, offset: np.ndarray, ) -> np.array:
```
Comment (Ipuch) [inline kinova_tracking/pos_kinova_arm_2.py:16]: int or str ??

index instead of indices

**Review by Ipuch — COMMENTED** [review]
Great I like it, Make sure to update the read me of models ;)

File: kinova_tracking/pos_kinova_arm_2.py (hunk 5 above)
Comment (AurelienRenou) [reply kinova_tracking/pos_kinova_arm_2.py:16]: It's a int

File: kinova_tracking/pos_kinova_arm_2.py (hunk 4 above)
Comment (AurelienRenou) [reply kinova_tracking/pos_kinova_arm_2.py:26]: it is, I forgot to modify the doc string

File: models/enums.py (hunk 2 above)
Comment (AurelienRenou) [reply models/enums.py:28]: done

File: models/enums.py (hunk 1 above)
Comment (AurelienRenou) [reply models/enums.py:20]: done

**Pull request merged**